vtkext/private/module/vtkF3DRenderer.cxx

```diff
@@ -274,7 +257,6 @@ void vtkF3DRenderer::Initialize()
   this->RenderPassesConfigured = false;
   this->LightIntensitiesConfigured = false;
   this->TextActorsConfigured = false;
-  this->MetaDataConfigured = false;
```

Comment:
why only this one ?

Reply:
that's a mistake, nice catch, I'll revert.